src/hooks/useConcentratorVaultTypeByAsset.ts

```diff
@@ -0,0 +1,24 @@
+import { Address } from "wagmi"
+
+import { VaultType } from "@/lib/types"
+import { useActiveChainId } from "@/hooks"
+
+const CONCENTRATOR_VAULT_TYPE_BY_ASSET: Record<
+  number,
+  Record<Address, VaultType>
+> = {
+  // arbitrum
+  42161: {},
+  // mainnet fork
+  31337: {},
+  // arbitrum fork
+  313371: {
+    "0xBDF9001c5d3fFc03AB6564CA28E530665594dfF7": "token",
```

Comment:
These same assets will apply to `arbitrum` too. Also, should we add these asset addresses to `constant/addresses.ts`?